### DISCUSSION TITLE
20:58:27 <ERROR : GameDepot> Relic properties are missing weights! Please check your ReliquaryMainPropExcelConfigData or ReliquaryAffixExcelConfigData files in your ExcelBinOutput folder.

### DISCUSSION THREAD

**Discussion by JieM-586245** [community/community#129856] [Programming Help]
### Select Topic Area

Question

### Body

![DAV7W$V}%VIVQAW 9N 9_ER](https://github.com/community/community/assets/172790223/d7609c61-5162-488d-8d52-8b3862b9d308)
这个错误怎么解决啊

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @JieM-586245 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
